src/applications/disability-benefits/rating-calculator/calculator-entry.js

```diff
@@ -0,0 +1,7 @@
+import './sass/disability-calculator.scss';
+
+import DisabilityRatingCalculator from './components/DisabilityRatingCalculator';
+
+export default {
+  DisabilityRatingCalculator,
+};
```

Comment:
It seemed redundant to include `disability` in this directory name since `disability-benefits` is its parent directory, but I included it anyway 🙂